src/clients/sw_single.rb

```diff
@@ -456,8 +456,9 @@ def GetPackagerOptions
         end
       end
       repo_management = Mode.normal if repo_management.nil?
+      online_search = Mode.normal && Yast::WFM.ClientExists("online_search")
```

Comment:
What if the system is not registered at all? How is that handled?

Reply:
It will be registered before registering the addons (see https://github.com/yast/yast-registration/pull/467/files#diff-57bedceb5faafb5d70b14b3357c09464R63).

Reply:
Please check also how it behaves in openSUSE Leap 15.1, you might create a reg code at https://scc.suse.com/admin/products/1929